lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -194,12 +194,12 @@ function dosomething_reportback_menu() {
     //'access callback' => 'dosomething_reportback_access',
     'access arguments' => array('administer modules'),
   );
-  $items['reportback/%reportback'] = array(
+  $items['admin/reportback/%reportback'] = array(
     'title' => 'Reportback',
-    'page callback' => 'dosomething_reportback_view_entity',
-    'page arguments' => array(1),
-    'access callback' => 'dosomething_reportback_access',
-    'access arguments' => array('view', 1),
+    'page callback' => 'dosomething_reportback_admin_view_entity',
+    'page arguments' => array(2),
+    // 'access callback' => 'dosomething_reportback_access',
+    'access arguments' => array('administer modules'),
```

Comment:
Campaigns team still need to access this view, and they don't have access to administer modules.  I believe changing the access arguments to `view any reportback` could do the trick...